lib/main_common.pm

```diff
@@ -2480,6 +2481,7 @@ sub set_mu_virt_vars {
     if (is_sle('15+')) {
         $scc_addons .= ',' if ($scc_addons);
         $scc_addons .= 'base,sdk,serverapp,desktop';
+        $scc_addons .= ',contm' if (get_var('KUBEVIRT_TEST'));
```

Comment:
how about development-tools-obs module?

Reply:
The development-tools-obs update repo will be added to INCIDENT_REPO , I think it is not needed to add it on host installation?

Reply:
I think it is also needed here. Update repo only contains updates. For pkgs without updates, they will be installed from module here.

Reply:
Above comment will be solved by existing development-tools-obs as add_on, plus a change suggested in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/21677/files#r2271924928.